misk/src/main/kotlin/misk/web/proxy/WebProxyAction.kt

```diff
@@ -99,3 +99,12 @@ class WebProxyAction @Inject constructor(
   }
 }
 
+/**
+ * https://github.com/google/guice/wiki/FrequentlyAskedQuestions#how-can-i-inject-optional-parameters-into-a-constructor
+ */
+@Singleton
+class OptionalProxyClient {
+  @Inject(optional = true)
```

Comment:
prefer OptionalBinder? https://groups.google.com/forum/#!topic/google-guice/zvhQiZozcDU

Reply:
yeah, I like that.